PWGJE/TableProducer/jettaggerhf.cxx

```diff
@@ -12,73 +12,129 @@
 // Task to produce a table joinable to the jet tables for hf jet tagging
 //
 /// \author Nima Zardoshti <nima.zardoshti@cern.ch>
+/// \author Hanseo Park <hanseo.park@cern.ch>
 
 #include "Framework/AnalysisTask.h"
 #include "Framework/AnalysisDataModel.h"
 #include "Framework/ASoA.h"
 #include "Framework/O2DatabasePDGPlugin.h"
 #include "Framework/runDataProcessing.h"
+#include "Common/Core/trackUtilities.h"
 
 #include "PWGJE/DataModel/Jet.h"
 #include "PWGJE/DataModel/JetTagging.h"
+#include "PWGJE/Core/FastJetUtilities.h"
 #include "PWGJE/Core/JetTaggingUtilities.h"
 #include "PWGJE/Core/JetDerivedDataUtilities.h"
 
 using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 
-template <typename JetTableData, typename JetTableMCD, typename JetTaggingTableData, typename JetTaggingTableMCD>
+template <typename JetTableData, typename JetTableMCD, typename JetTaggingTableData, typename StoredJetTaggingTableData, typename JetTaggingTableMCD, typename StoredJetTaggingTableMCD>
 struct JetTaggerHFTask {
 
   Produces<JetTaggingTableData> taggingTableData;
+  Produces<StoredJetTaggingTableData> storedTaggingTableData;
```

Comment:
I think you can change the name of this to something else. Since both this and the one above it are stored. Also the logic seems a bit strange. You have the algorithm outputs in jettaggingtabledata and then you have some variable outputs in storedjettaggingtabledata. However it seems these should be two different tasks and that you should be able to run the trained ml model in this task